mrtarget.yml

```diff
@@ -0,0 +1,90 @@
+
+#reactome
+reactome-pathway-data: https://storage.googleapis.com/open-targets-data-releases/19.02/input/annotation-files/ReactomePathways-2019-01-09.txt
+reactome-pathway-relation: https://storage.googleapis.com/open-targets-data-releases/19.02/input/annotation-files/ReactomePathwaysRelation-2019-01-09.txt
+
+#uniprot
+uniprot-uri: https://storage.googleapis.com/ot-releases/18.12/annotations/uniprot-20181130.xml.gz
+
+#gene plugins
+gene-data-plugin-places: [mrtarget/plugins/gene]
+gene-data-plugin-names: [HGNC,Orthologs,Ensembl,Uniprot,ChEMBL,MousePhenotypes,Hallmarks,CancerBiomarkers,ChemicalProbes,Tractability]
+
+#hgnc gene plugin
+#TODO move this to GCP https://github.com/opentargets/platform/issues/326
+hgnc-complete-set: http://ftp.ebi.ac.uk/pub/databases/genenames/new/json/hgnc_complete_set.json
+
+#ortholog gene plugin
+#TODO move this to GCP https://github.com/opentargets/platform/issues/326
+hgnc-orthologs: http://ftp.ebi.ac.uk/pub/databases/genenames/hcop/human_all_hcop_sixteen_column.txt.gz
+hgnc-orthologs-species: [9606-human,9598-chimpanzee,9544-macaque,10090-mouse,10116-rat,9615-dog,9823-pig,8364-frog,7955-zebrafish,7227-fly,6239-worm,4932-yeast]
+
+#mouse phenotype plugin
+#TODO move this to GCP https://github.com/opentargets/platform/issues/379
+mouse-phenotypes-orthology: http://www.informatics.jax.org/downloads/reports/HMD_HumanPhenotype.rpt
+#TODO move this to GCP https://github.com/opentargets/platform/issues/379
+mouse-phenotypes-report: http://www.informatics.jax.org/downloads/reports/MGI_PhenoGenoMP.rpt
```

Comment:
The same for ortholog and mouse phenotype files - will they be added to the annotations folder?